src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -19,6 +19,11 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="AllocationHelper.cs" />
+    <Compile Include="Span\SequenceCompareTo.bool.cs" />
```

Comment:
nit: Please sort these alphabetically and move them to the ItemGroup with the rest of the Span tests (here: https://github.com/dotnet/corefx/blob/5f4e97d668677210cb41acf9f0df0e2d71382cb5/src/System.Memory/tests/System.Memory.Tests.csproj#L63)